recipe/migrations/cuda120.yaml

```
@@ -5,7 +5,7 @@ __migrator:
   migration_number:
     1
   build_number:
-    1
+    2
```

Think this is just for the build number increment in PRs. Would it be alright if we kept this?

```suggestion
    1
```

---

🤷🏼 I am honestly not sure what this parameter does.